server/services/sessionTemplateService.ts

```diff
@@ -24,4 +25,12 @@ export default class SessionTemplateService {
 
     return visitSchedulerApiClient.getSingleSessionTemplate(reference)
   }
+
+  async createSessionTemplate(username: string, sessionTemplate: CreateSessionTemplateDto): Promise<void> {
+    const token = await this.hmppsAuthClient.getSystemClientToken(username)
+    const visitSchedulerApiClient = this.visitSchedulerApiClientFactory(token)
+
+    logger.info(`Adding session template ${sessionTemplate.name} to prison ${sessionTemplate.prisonId}`)
```

Comment:
Probably better to capture the created template and then we can log its actual reference.(and if it fails, this will end up in logs anyway)